### PR TITLE
Added us/pa/york

### LINKED ISSUE #1977: Add addresses in York, PA
Status:
- [ ] **Research sources** and propose best source (and it's data license)
- [ ] **Review proposed source**
- [ ] **Approve source** based on review comments
- [ ] **Add** to OpenAddresses

_NOTE: Proposed source notes should include basic column mapping to OpenAddress data model._

### PR DESCRIPTION
**Pull request by davidchiles**
Fixes #1977.

### CODE REVIEW

**Comment by trescube** [comment]
Fantastic!  My home county, I had been looking for this data for awhile.

**Comment by davidchiles** [comment]
re-run this please

**Comment by davidchiles** [comment]
Looks like there was a time out on some of the requests.
Not sure I can do this but I'll try...
re-run this, please

File: sources/us/pa/york.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "conform": {
+        "type": "geojson",
+        "number": "H_NUMBER",
+        "street": [
+            "ST_DIR",
+            "STREET_NAME",
+            "ST_TYPE",
+            "ST_SUFFIX"
+        ], 
+        "unit": "UNIT_NUMBER",
```
Comment (trescube) [inline sources/us/pa/york.json]: please add `UNITTYPE` from the source here

Comment (davidchiles) [inline sources/us/pa/york.json]: Would it be better combined in `unit` or as `addrtype`?

```js
"unit": ["UNITTYPE", "UNIT_NUMBER"]
```
or 
```js
"addrtype" : "UNITTYPE"

Comment (trescube) [inline sources/us/pa/york.json]: Traditionally it's been the former.

**Comment by trescube** [comment]
There's some messy data in there but a great addition overall!

**Pull request merged**